docs/source/utils.rst

```diff
@@ -7,7 +7,6 @@ Module with helper methods
 
 .. autosummary::
     :nosignatures:
-    :autolist:
```

Comment:
Why do you remove this, John?

Reply:
`make clean; make html` fails if that line is added back in and succeeds when it is removed. I had another look at it there and have pushed a fix for the AutolistAutosummary class so I don't need to remove this line any more.